.gitignore

```diff
@@ -127,3 +127,4 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+.DS_Store
```

Comment:
pelase undo this modificaiton, you have modified seven files, you should change only 2